code/modules/mob/living/carbon/xenomorph/update_icons.dm

```diff
@@ -52,14 +52,23 @@
 		return
 
 	if(stat == DEAD)
-		icon_state = "[mutation_type] [caste.caste_type] Dead"
+		icon_state = "[mutation_icon_state || mutation_type] [caste.caste_type] Dead"
+		if(!icon state) in icon_states(icon_xeno)
```

Comment:
```suggestion
		if(!icon_state) in icon_states(icon_xeno)
```

Reply:
I had to go into VSC to figure out what I was looking at.